src/collector.ts

```diff
@@ -58,51 +51,61 @@ export async function runCollector(
       };
       sendEvent(worker, event);
     }
-    if (ledgersProcessed % 60 === 0) {
-      // approx every 5m
-      // send an oracle scan event
-      const event: OracleScanEvent = {
-        type: EventType.ORACLE_SCAN,
-        timestamp: Date.now(),
-      };
-      sendEvent(worker, event);
-    }
-    if (ledgersProcessed % 1203 === 0) {
-      // approx every 2hr
-      // send a user update event to update any users that have not been updated in ~2 weeks
-      const event: UserRefreshEvent = {
-        type: EventType.USER_REFRESH,
-        timestamp: Date.now(),
-        cutoff: Math.max(latestLedger - 14 * 17280, 0),
-      };
-      sendEvent(worker, event);
-    }
-    if (ledgersProcessed % 1207 === 0) {
-      // approx every 2hr
-      // send a liq scan event
-      const event: LiqScanEvent = {
-        type: EventType.LIQ_SCAN,
+
+    for (const poolConfig of poolConfigs) {
+      // new ledger detected
+      const ledger_event: LedgerEvent = {
+        type: EventType.LEDGER,
         timestamp: Date.now(),
+        ledger: latestLedger,
+        poolConfig,
       };
-      sendEvent(worker, event);
+      sendEvent(bidder, ledger_event);
```

Comment:
IMO ledger events should still be just per ledger, not sent per pool, this will make bidder logs extremely chatty, which are the most important logs in the system.

The bidder logic manages 1 submission queue, and 1 db table with ongoing auctions. It can infer the pool from the ongoing auction it is managing, or the submission it is monitoring.